pycbc/live/snr_optimizer.py

```diff
@@ -256,17 +248,15 @@ def optimize_pso(bounds, cli_args, extra_args, initial_point):
         bounds['spin2z'][1]
     ])
 
-    # Manually generate the initial points, this is the same as the default
-    # method, but allows us to make some modifications
-    population = numpy.random.uniform(
-        low=0.0, high=1.0, size=(int(cli_args.snr_opt_pso_particles), 4)
```

Comment:
I made a suggestion for the formatting here, but you also have introduced a bug because the size here was controlled by the command line argument for PSO, but you have broken that.